duckduckgo/iOS
# Subscription-related updates

**Description**:
- Cherry-picking subscription-related updates from main

Steps to test this PR:

1. Smoke test subscriptions

2. Test Native redirects
- Go to any website
- Go to https://sbarag.duckduckgo.com/pro
- Go back until you get to the browser, you should be on the first page you loaded
- Open a new tab and go to https://codepen.io/marcosholgado/pen/PogEJrM
- Click on Click Here
- You should be taken to the native flow
- Go back to the browser, you should be in the same page as before
- Long tap on click here and open in a new tab
- You should be taken to the native flow
- Go back to the browser, you should be in the same page
- Long click on click here and open in a new background tab
- You should be taken to the native flow
